compiler/rustc_codegen_llvm/src/context.rs

```diff
@@ -660,6 +671,11 @@ impl<'ll, CX: Borrow<SCx<'ll>>> GenericCx<'ll, CX> {
         llvm::LLVMMetadataAsValue(self.llcx(), metadata)
     }
 
+    pub(crate) fn get_const_i64(&self, n: u64) -> &'ll Value {
```

Comment:
leave a FIXME that we should split `ConstCodegenMethods` to pull the reusable parts onto a trait that is also implemented for `GenericCx`

Reply:
Agree, I tried that and failed, but my attempt was probably too naive.